src/ut/handlers_test.cpp

```diff
@@ -95,6 +95,7 @@ class HandlersTest : public testing::Test
   static const std::string DEST_HOST;
   static const std::string DEFAULT_SERVER_NAME;
   static const std::string SERVER_NAME;
+  static const std::string WILDCARD;
```

Comment:
Change from "wildcard" to "wildcarded_public_identity" - same change needed in lots of places.